transports/sse/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```
@@ -0,0 +1,8 @@
+name: Model Context Protocol (MCP) Server for server side events (SSE)
+metadata:
+  keywords:
+    - mcp-server
+  guide: https://docs.quarkiverse.io/quarkus-mcp-server/dev/index.html # To create and publish this guide, see https://github.com/quarkiverse/quarkiverse/wiki#documenting-your-extension
```

```suggestion
  guide: https://docs.quarkiverse.io/quarkus-mcp-server/dev/index.html
```

This guide is already published...